### PR TITLE
Tagging example

### PR DESCRIPTION
**Pull request by diicellman**
# Description

Tagging system. It's designed to tag input text, the output of the tagging process is structured using Pydantic. And it can also operate locally via Ollama.

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Added new unit/integration tests
- [ ] Added new notebook (that tests end-to-end)
- [ ] I stared at the code and made sure it makes sense

# Suggested Checklist:

- [ ] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added Google Colab support for the newly added notebooks.
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I ran `make format; make lint` to appease the lint gods

### CODE REVIEW

File: examples/tagging/data/test_tweets_anuFYb8.csv
Comment (hatianzhang) [inline examples/tagging/data/test_tweets_anuFYb8.csv:1]: better not to upload such data.
can you put this data into a google drive and share the link for the notebook to download

File: examples/tagging/private_tagging.ipynb [hunk 1]
```diff
@@ -0,0 +1,192 @@
+{
```
Comment (hatianzhang) [inline examples/tagging/private_tagging.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    mistral_output = mistral_program(user_comment="I hate this people they're lazy as fuck!")
</pre><p>emm could we use a better wording lol</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (diicellman) [reply examples/tagging/private_tagging.ipynb:1]: Okay, I'll change the phrasing to more appropriate one 😅

**Review by hatianzhang — APPROVED** [review]
LGTM, just fixed some lint issues

**Pull request merged**